README.md

```diff
@@ -11,6 +12,59 @@ You can reach the maintainers of this project at:
 - [Slack channel](https://kubernetes.slack.com/messages/sig-node)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
 
+ 
+## Building the operator
+
+Checkout the sources
+
+```bash
+$ git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+
+Build the operator image
+
+```bash
+make image IMAGE=<my repo>:<my tag>
+```
+
+Optionally you can push it to your image repo 
+
+```bash
+make image-pus IMAGE=<my repo>:<my tag>
+```
+
+## Manual deploy of the operator
+
+Update the  `Makefile` with the custom image built and configure the namespace where the operator should be deployed.
+
+The default CR will create the operand (NFD) in the `node-feature-discovery-operator` namespace,
+the CR can be edited to choose another namespace and image. See the `manifests/0700_cr.yaml` for the default values.
+
+```bash
+$ make deploy
+```
```

Comment:
nit: I would be consistent with the building instructions above i.e. instruct to do
```bash
$ make deploy IMAGE=<my repo>:<my tag>
```
like in the building instructions, or, suggest to edit the Makefile in both places.

Just occurred to me that `IMAGE` can also be specified in the environment so one more alternative way would
```bash
export IMAGE=<my repo>:<my tag>
make image
```
and then omit the "IMAGE" part from the later instructions. WDYT?

Reply:
I prefer explicit declarations of Env Vars, some users may have weird bash sessions and some times make won't take the env var. 

I will go for 
```bash
$ make deploy IMAGE=<my repo>:<my tag>
```